crates/primitives/starknet/src/execution/call_entrypoint_wrapper.rs

```diff
@@ -162,7 +161,7 @@ impl TryInto<CallEntryPoint> for CallEntryPointWrapper {
             // starknet-lib is constantly breaking it's api
             // I hope it's nothing important ¯\_(ツ)_/¯
             code_address: None,
-            initial_gas: Felt252::from_bytes_be(&self.initial_gas.0.to_bytes_be()),
+            initial_gas: self.initial_gas.try_into().unwrap(),
```

Comment:
Why is this unwrap safe?

Reply:
So we have initial_gas set to 0 everywhere which should be fixed over here #710. However, even after that, `initial_gas` should be less than u64 and if it's not I feel that error should be caught somewhere at the RPC level. Should I handle it here as well?

Reply:
No need to handle the error if it cannot happen. But you need to explain why on a comment above the unwrap.

Reply:
So just say what you tlod me in a comment

Reply:
done!